### PR TITLE
expose User Contact in standaloneusers UI

### PR DESCRIPTION
**Pull request by ufundo**
Overview
----------------------------------------
Exposes the Contact ID for a Standalone User in the UI - means you can see and edit through the Administer pages

Before
----------------------------------------
Setting and updating user contact ID is API only.

After
----------------------------------------
Linked contact is displayed on the user listing, can be set on user create, and edited on user edit.

Contact ID is required -- this seemed good for now at least because logging in as a user with no contact ID crashes things. I did wonder if there could be use-cases for Users without Contacts (service accounts?) but I think it's assumed in a lot of places atm so that would be more work.

Comments
----------------------------------------
Editing the contact record doesn't impact the login email for the user account. Think we concluded this was the right way to go in discussion this morning -- you don't want to non-deliberately change admin user email.

### CODE REVIEW

**Comment by ufundo** [comment]
Just added ability to quick add a new contact on the contact selector

**Comment by ufundo** [comment]
Nothing here to restrict user contacts to Individuals - i.e. you could have User for an Organisation. Seems reasonable to me but is this allowed behaviour elsewhere? :thinking:

**Comment by ufundo** [comment]
hi @artfulrobot I've updated to your branch now :+1: 

would be happy to restrict to Individuals but not sure where is the best place to do that - in the BAO maybe? I think it could be a separate PR or even a setting?

File: ext/standaloneusers/xml/schema/CRM/Standaloneusers/User.xml [hunk 1]
```diff
@@ -82,6 +82,7 @@
     <comment>FK to Contact ID</comment>
     <html>
       <label>Contact</label>
+      <type>EntityRef</type>
```
Comment (colemanw) [inline ext/standaloneusers/xml/schema/CRM/Standaloneusers/User.xml:85]: You need to put Individual *here:* 😆 
```suggestion
      <type>EntityRef</type>
      <filter>contact_type=Individual</filter>
```

**Comment by colemanw** [comment]
@ufundo apparently this conflicted with your other PR and needs to be rebased again...

**Comment by ufundo** [comment]
Thanks both :grin:

**Pull request merged**